### ISSUE TITLE
Reproduced mAP on AITODv2 (28.4) lower than paper reported (30.5)

### ISSUE THREAD

**Issue by REREREGO** [hoiliu-0801/DQ-DETR#36]
Hi, thank you for the great work and for open-sourcing the code.
I tried to reproduce the results on the AITODv2 dataset following the default configuration, but my reproduced mAP is 28.4, which is lower than the 30.5 reported in the paper.
Could you please help clarify a few details?
- Training settings: Did you use any specific hyperparameters or training tricks beyond the default config (e.g., learning rate schedule, data augmentation, multi-scale training, number of GPUs/batch size)?
- Randomness: Did you observe significant variance across different random seeds?       

Any guidance would be appreciated. Thanks!

<img width="1017" height="671" alt="Image" src="https://github.com/user-attachments/assets/f2cc2994-897e-4a10-85bb-3b742aa60681" />

**Comment by superbigyamy-byte** [comment]
Hello, I see that you successfully ran the code. Do you happen to still have the files `pretrain_model.pth` and `dqdetr_best305.pth`? The links provided by the author are no longer working.

**Comment by REREREGO** [comment]
额我之前放弃这块的实验了，相关的代码和checkpoint也给我删了，可能没办法帮到你了，抱歉




嘻嘻哈哈
***@***.***
